app/views/candidate_mailer/conditions_met.text.erb

```diff
@@ -1,9 +1,19 @@
 Dear <%= @application_form.first_name %>
 <% if @application_form.show_new_reference_flow? %>
 
-  <%= @course.provider.name %> has confirmed that you’ve met your conditions and they’ve checked your references.
+  <%= @course.provider.name %> has confirmed that you’ve met the conditions of your offer.
 
-  If they have not already, <%= @course.provider.name %> should contact you with details about enrolling on the course. Contact them directly if you have any questions.
+  They’ll let you know if they need further information before you can start training. Contact them if you have any questions.
+
+  You can sign into your account if you need to check the progress of your reference requests.
+
+  [Sign into your account](<%= candidate_magic_link(@application_form.candidate) %>).
+
+  #If you’re unable to start training in <%= @start_date %>
```

Comment:
Is this meant to be a subheading? Think it needs two `##` and a space?
```suggestion
## If you’re unable to start training in <%= @start_date %>
```

Reply:
I used the same syntax as app/views/candidate_mailer/unconditional_offer_accepted.text.erb, which currently renders correctly as a heading.

Reply:
Interesting! Looks like notify does its own thing with markdown syntax...